pkg/operator/controller/nodeobservability/daemonset.go

```diff
@@ -145,8 +145,9 @@ func (r *NodeObservabilityReconciler) desiredDaemonSet(nodeObs *v1alpha1.NodeObs
 							},
 						},
 						{
-							Name:  "kube-rbac-proxy",
-							Image: "gcr.io/kubebuilder/kube-rbac-proxy:v0.11.0",
+							Name:            "kube-rbac-proxy",
+							Image:           "gcr.io/kubebuilder/kube-rbac-proxy:v0.11.0",
+							ImagePullPolicy: corev1.PullAlways,
```

Comment:
What is the reason we need to modify the default policy?

Reply:
I changed this for testing - so I have reverted back to PullIfNotPresent